tests/testthat/_snaps/constructors.md

```diff
@@ -145,26 +145,25 @@
 
     Code
       mtry()
-    Output
+    Message
       # Randomly Selected Predictors (quantitative)
       Range: [1, ?]
 
 ---
 
     Code
       surv_dist()
-    Output
-      Distribution  (qualitative)
     Message
+      Distribution (qualitative)
       6 possible values include:
-    Output
-      'weibull', 'exponential', 'gaussian', 'logistic', 'lognormal' and 'loglogistic' 
```

Comment:
notice here how before it was a mix of `Output` and `Message`, but now it is just `Message`. Because now we no longer mix and match {cli} with `cat()`